src/matrix.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * OpenSeadragon - Mat3
+ *
+ * Modified from https://webglfundamentals.org/webgl/lessons/webgl-2d-matrices.html
```

Comment:
I'm not sure the best way to provide the proper license here. For reference, I believe this is the WebGL Fundamentals license: 

https://github.com/jiebai/webgl-fundamentals/blob/master/LICENSE

On the other hand, WebGL Fundamentals is explicitly a teaching site, so maybe its example code is free to use? Probably best not to assume that...

Reply:
Probably best is to follow what we've already done for jQuery:

https://github.com/openseadragon/openseadragon/blob/6abcc76a93ee3369267908f0a80641dc171440ed/src/openseadragon.js#L36

Reply:
I've tried to recreate the style of the jQuery attribution and licensing - let me know if you think it should be modified further